cutadapt.wdl

```diff
@@ -151,4 +151,239 @@ task Cutadapt {
         memory: memory
         docker: dockerImage
     }
+
+    parameter_meta {
+        read1: {
+            description: "The first or single end FastQ file to be run through cutadapt.",
+            category: "required"
+        }
+        read2: {
+            description: "An optional second end FastQ file to be run through cutadapt.",
+            category: "common"
+        }
+        read1output: {
+            description: "The name of the resulting first or single end FastQ file.",
+            category: "common"
+        }
+        read2output: {
+            description: "The name of the resulting second end FastQ file.",
+            category: "common"
+        }
+        format: {
+            description: "Equivalent to cutadapt's --format option.",
+            category: "advanced"
+        }
+        adapter: {
+            description: "A list of 3' ligated adapter sequences to be cut from the given first or single end fastq file.",
```

Comment:
FastQ and fastq are both used in this document, would it be nicer to just use one?